frappe/database/utils.py

```diff
@@ -34,6 +34,14 @@ def is_pypika_function_object(field: str) -> bool:
 	return getattr(field, "__module__", None) == "pypika.functions" or isinstance(field, Function)
 
 
+def get_doctype_name(table_name: str) -> str:
+	if "tab" in table_name:
+		table_name = table_name.replace("tab", "")
```

Comment:
tab can be part of doctype name too. 

![image](https://user-images.githubusercontent.com/9079960/212833932-1e76d6af-d761-44c3-a789-517fef213964.png)